backend/core/src/Core.Application/Queries/CustomerQueries/GetCustomerQuery.cs

```diff
@@ -33,11 +33,6 @@ public async Task<Customer> Handle(GetCustomerQuery request, CancellationToken c
         {
             var customer = await _customerRepository.GetAsync(request.CustomerCode);
 
-            if (customer.Status != CustomerStatus.ACTIVE.ToString() && customer.Status != CustomerStatus.UNDERREVIEW.ToString())
```

Comment:
I see there are now two unused usings (`Core.Domain` and `Core.Domain.Exceptions`) with the removal of them. Could you also remove these please :)

Reply:
done